packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-spread/assignment/expected.js

```diff
@@ -1,5 +1,5 @@
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
-z = _extends({ x }, y);
+z = _extends({}, { x }, y);
```

Comment:
Since this is the actual:
```js
z = { x, ...y };

z = { x, w: { ...y } };
```

I think this change isn't needed. However, with this actual:
```js
z = { get x() {}, ...y };
```
I'm not sure if it should be or not (https://github.com/tc39/proposal-object-rest-spread/issues/46)